src/focus.service.ts

```diff
@@ -421,10 +380,17 @@ export class FocusService {
 
   public createArcEvent(direction: Direction): ArcEvent {
     const directional = isDirectional(direction);
+    let element: HTMLElement;
+    if (directional) {
+      element = this.findNextFocusByRaycast(direction);
+      if (!element) {
+        element = this.findNextFocus(direction);
```

Comment:
The idea is to have different focus state for keyboard nav vs point&click av
https://github.com/WatchBeam/ui/blob/81cc92782fdc64c21f42930bd27eb4cb1f774e32/src/lib/button/button.scss#L47